path_fetch.go

```diff
@@ -1,48 +1,26 @@
 package keyfactor
 
-import (
```

Comment:
I think this functionality was moved to `path_ca.go` should we just delete this file?